web/src/components/Plugin/UI/cors.tsx

```diff
@@ -156,15 +161,15 @@ const Cors: React.FC<Props> = ({ form }) => {
               ))}
               {
                 <Form.Item {...FORM_ITEM_WITHOUT_LABEL}>
-                  <Button
+                  {fields.length < maxLength? (<Button
```

Comment:
`{fields.length < maxLength && <Button>...</Button>}`

Reply:
Thanks for your suggestion, I have modified it